src/expandableCalendar/week.tsx

```diff
@@ -7,27 +7,48 @@ import {parseDate, toMarkingFormat} from '../interface';
 import {getState} from '../day-state-manager';
 import {extractDayProps} from '../componentUpdater';
 import styleConstructor from './style';
-import Calendar, {CalendarProps} from '../calendar';
+import {CalendarProps} from '../calendar';
 import Day from '../calendar/day/index';
-// import BasicDay from '../calendar/day/basic';
+import {CalendarContextProps} from './Context';
+import {isEmpty, isEqual} from 'lodash';
 
 
-export type WeekProps = CalendarProps;
+export type WeekProps = CalendarProps & {
+  context?: CalendarContextProps;
+};
+
+const propsAreEqual = (prevProps: WeekProps, nextProps: WeekProps) => {
+  if (isEmpty(nextProps.context)) {
```

Comment:
This is blocking updates from the compoent when it is not use from the WeekCalendar, meaning, when there is no `context` passed. In the case of ExpandableCalendar, passing `disableWeekScroll` prop will render the Week component directly and it will not work properly - see my comment above.